Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Start with a base image
+FROM python:3-onbuild as base
+# FROM apache/airflow:master-python3.6-ci
+# FROM python:3.6-slim-buster
+# FROM continuumio/miniconda3
+
+ARG DB_HOST=localhost
+ARG DB_PORT=5432
+ARG DB_NAME=postgres
+ARG DB_USER=postgres
+ARG DB_PASS
+
+ENV orion_prod=postgres+psycopg2://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/${DB_NAME}
+ENV orion_test=postgres+psycopg2://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/postgres
+ENV airflow_db=postgres+psycopg2://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/airflow
+
+ENV AIRFLOW_HOME /airflow/orion/core
+ENV AIRFLOW__CORE__SQL_ALCHEMY_CONN ${airflow_db}
+ENV AIRFLOW__CORE__EXECUTOR LocalExecutor
+ENV AIRFLOW__CORE__DAGS_FOLDER /airflow/orion/core/dags
+
+FROM base as builder
+
+RUN mkdir /install
+WORKDIR /install
+
+COPY requirements.txt /requirements.txt
+
+RUN pip install --upgrade pip \
+  && pip install --install-option="--prefix=/install" -r /requirements.txt
+
+FROM base
+
+WORKDIR /airflow
+
+COPY --from=builder /install /usr/local
+COPY . ./
+COPY entrypoint.sh /entrypoint.sh
+COPY boto.cfg /etc/boto.cfg
+
+
+RUN echo $orion_prod \
```

Comment:
What's the purpose of `echo $orion_prod`? It reveals AWS secrets so we should delete it if it's not needed.

Reply:
Forgot it there — was for debugging purposes :)